Observing-Interface-Setup/dds-interface.rst

```diff
@@ -0,0 +1,28 @@
+#############
+Debugging DDS
+#############
+
+Overview
+^^^^^^^^
+This topic describes procedures for debugging the status of the `DDS interface <https://ts-sal.lsst.io>`_ that powers the :doc:`CSCs <obs-controls:System-Architecture/CSC-Overview/index>` that run on the summit.
+
+Checking the status of the Daemon
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
It's worth a couple sentences here saying when this is applicable. An example of a demon failure (with a screenshot) might be useful.

Reply:
Uhm... this is very site and environment specific as well... need more context about when this is applicable.